components/p3a/brave_p3a_service.cc

```diff
@@ -85,6 +96,32 @@ base::Time NextMonday(base::Time time) {
   return result;
 }
 
+base::Time NextDay(base::Time time) {
+  return (time.LocalMidnight() + base::Days(1) + base::Hours(4))
```

Comment:
Is there a reason for `base::Hours(4)`?

Reply:
`time.LocalMidnight + base::Days(1)` should give you 00:00:00 on the next day

Reply:
Please see comment on line 92

Reply:
I'd duplicate the comment :)